utils/_context/_scenarios/core.py

```diff
@@ -317,6 +317,8 @@ def _get_warmups(self):
 
         return warmups
 
+    # Build a tree of dependencies between containers and then start the
+    # containers in the right order and in parallel if possible.
     def _start_containers(self):
```

Comment:
Python stuff here 

```
    def _start_containers(self):
    """ 
      Build a tree of dependencies between containers and then start the
      containers in the right order and in parallel if possible.
    """
```

it's what's called docstrings, it'll be attached to the method, and for IDE that supports python, will. be displayed on hoverging the function calls.

Same review for other comments